assets/model_monitoring/components/src/model_monitor_create_manifest/run.py

```diff
@@ -27,10 +27,19 @@ def _generate_manifest(root_dir: str):
 
 def run():
     """Create Manifest."""
-    # init aml OBO class for supporting credential-less datastore scenarios.
-    # this class will init specific env variables required by Amlfs to get
-    # user token in case of credential-less data access scenario
-    AzureMLOnBehalfOfCredential()
+    # TODO: investigate why these aren't initialized by AzureML Filesystem itself
+    # init env-vars for supporting credential-less datastore scenarios.
+    # this env variable is required by Amlfs to get user token.
+    spark = init_spark()
```

Comment:
we replace AmlOBOCredential() with below env var setup code? You copy some initialization code from there?

Reply:
Yes I grabbed this code from the internals of AML obo class. If we want to avoid the import issue caused by old cache this is the only way to still support the credential-less save scenario on this component

Reply:
In reality this is probably a simpler and clearer way than having the adhoc aml obo credential instantiation.

Reply:
consider to make it a util function